spec/main_spec.rb

```diff
@@ -0,0 +1,55 @@
+require '../lib/board.rb'
+require '../lib/player.rb'
+describe Board do
```

Comment:
[OPTIONAL]
Consider having `Board` and `Player` tests in separate files like `board_spec.rb` and `player_spec.rb` for separation of concerns.

But seeing as these tests are not lengthy, this is fine. But it's good practice to separate. 🙂